service/history/queue/timer_queue_processor.go

```diff
@@ -178,6 +178,8 @@ func (t *timerQueueProcessor) Stop() {
 
 	if !t.shard.GetConfig().QueueProcessorEnableGracefulSyncShutdown() {
 		t.activeQueueProcessor.Stop()
+		// stop active executor after queue processor
+		t.activeTaskExecutor.Stop()
```

Comment:
There are also `standbyTaskExecutor` objects created and passed to `newTimerQueueStandbyProcessor` in the constructor. We need to stop them as well.  Timer queue processor is creating those so it should be responsible of stopping the executors

Reply:
Thanks, I have added a field accounting for the standbyTaskExecutors and stop then in the end.